### DISCUSSION TITLE
Firefox ESR releases

### DISCUSSION THREAD

**Discussion by maederm** [microsoft/winget-pkgs#82511] [Q&A] [answered]
Hi

Should I always get the newest Firefox ESR when using winget-pkgs? Currently I get 102.0, the most recent version is 102.3.0.
Is this a manual process? I could see in issue #500 that an automation is planned but it seems to be open.

Cheers

**Comment by mdanish-kh** [comment, marked as answer]
You should get the version whose manifest is submitted and merged into the repository. Currently this is a manual process for `Firefox` and many other packages - the `Firefox` manifests are usually updated by @jedieaston.

**Reply by jedieaston** [reply]
Yep. I need to update ESR — thanks for tagging me
